### ISSUE TITLE
add county boundaries overlay

### ISSUE THREAD

**Issue by rgdonohue** [chrispappas007/kyarch#10]
Might help users locate areas of interest

**Comment by chrispappas007** [comment]
So should the county data just capture how many hexagons fall within that county, or create a county layer with the totals already included?

**Comment by rgdonohue** [comment]
Oh, you had mentioned adding a counties border layer. I was just thinking thin outlines of the counties somehow, with no data. But only if it doesn't become to visually complicated with the hexagons.

**Comment by chrispappas007** [comment]
Yes, I definitely want to have the county outlines, but I was thinking it could be a layer that is toggled on/off. When toggled on, you could click on the county and see the number of sites of that selected time period within that county and the total number of sites for that county. It's easy enough to add a column of total sites for each county to the county data and then maybe just have it where when the county layer is toggled on, it selects the hexs within the county and gives the total for the selected period with the total for the entire county.

**Comment by rgdonohue** [comment]
That sounds good. Of course, then you're only a couple steps away from having that toggled county layer be a choropleth as well (normalized by land area)?

**Comment by chrispappas007** [comment]
Ugh! Why did you go there? Now I have to think about that too and test it out! 

I like the idea of the information that will provide, but then it's a matter of the visual impact. The best way then may be to toggle between the hex grid and the county data. If the hexs just site on a county choropleth, then it may muddy the data. I think the options are to either have the county data just be a thin outline of counties (maybe toggle on and off? Definitely toggle-able if the sum data is provided) or have the county data be a choropleth and toggle with the hexs. 

I need to think on this...

**Comment by chrispappas007** [comment]
Counties are loaded as a background layer, but no other UI as of now. I want to get the picture/modal sorted first and then worry about this crazy county choropleth option. I synced the code to the repo so have a look and let me know your thoughts on counties as just the outlines with no UI. I also think I need to find a different basemap too.
